tests/kola/version/rhel-major-version

```diff
@@ -12,8 +12,11 @@ fatal() {
 source /etc/os-release
 
 
+RHEL_VERSION="8"   
```

Comment:
You used source and then replaced the value for 8. Remove this line or it will always be 8